grouper/fe/templates/email/permission_request_cancelled_html.tmpl

```diff
@@ -0,0 +1,19 @@
+{% extends "email/base_html.tmpl" %}
+
+{% block subject %}Request for Permission Cancelled{% endblock %}
```

Comment:
Nit: You could probably combine this with the actioned template with a bit more templating.  Not sure if it's worth it.